src/test/java/com/wikia/webdriver/elements/oasis/components/notifications/NotificationType.java

```diff
@@ -0,0 +1,9 @@
+package com.wikia.webdriver.elements.oasis.components.notifications;
+
+public enum NotificationType {
```

Comment:
You can restrict types used in methods to the ones from enum not to let people pass raw strings